modules/darwin/common/secrets.yaml

```diff
@@ -0,0 +1,57 @@
+cachix-agent.token: ENC[AES256_GCM,data:BiRRAIw5A76oBdO+YWR0icFS4s3AbXuHWj1R9LTCJ7N4CF7qaH89NKwXEchfwEShJNay1vG3K/jtpaigwoYaEDmgj1YrEUBq3Tne17S8d4AzBr+s2FiOA0iv7T6/szcMm5ShspKl1xYu70mZDxcEuuEI0So8IBq1x2brB5Edw4tN39XrsXKUVIyvODJHQjSyEn0yJOuLw+0FbLZJvt27FQiXoMXoyW0jLh+1NbXY8C1CIg==,iv:8GIZzHaF7mbXOKfSq3vBc4wGa7NUZKbeLNIVxWqiBhg=,tag:HkeJmqVe+Yzf5C+EuF9m2A==,type:str]
```

Comment:
I've generated a separate token for darwin.

It's been installed to the default path `/etc/cachix-agent.token` on both darwin machines.